sycl/source/detail/scheduler/commands.cpp

```diff
@@ -82,6 +90,37 @@ static std::string accessModeToString(access::mode Mode) {
   }
 }
 
+#ifdef XPTI_ENABLE_INSTRUMENTATION
+// Using the names being generated and the string are 
+// subject to change to something more meaningful to 
+// end-userws as this will be visible in analysis tools
+// that subscribe to this data
+static std::string commandTypeToString(Command::CommandType type) {
+  switch(type) {
+    case Command::CommandType::RUN_CG:
+       return "RUN_CG";
+    case Command::CommandType::COPY_MEMORY:
+       return "COPY_MEMORY";
+    case Command::CommandType::ALLOCA:
```

Comment:
I am currently using this function, because it exists. You will see this strings show up in the telemetry data and I am not really fond of it. Need to map to meaningful strings at some point.